Diagnostic/diagnostic.py

```diff
@@ -238,11 +238,10 @@ def encrypt_string(cert, secret):
     global enable_metrics_ext
```

Comment:
is this still being used

Reply:
Yeah it is used down in the main and start_mdsd methods